cmd/bosun/sched/host.go

```diff
@@ -440,6 +440,24 @@ func (s *Schedule) Host(filter string) (map[string]*HostData, error) {
 					if iface != nil {
 						iface.Description = val
 					}
+				case "dataStores":
+					dataStores := []string{}
+					err = json.Unmarshal([]byte(val), &dataStores)
+					if err != nil {
+						slog.Errorf("error unmarshalling datastores for host %s while generating host api: %s", host.Name, err)
```

Comment:
It is possible it has a few partial objects already created there. I'd rather skip it to be safe.